qp_qiime2/__init__.py

```diff
@@ -21,176 +25,179 @@
-         '"margalef", "lladser_ci", "pielou_e", "michaelis_menten_fit", '
-         '"ace", "chao1_ci", "goods_coverage", "esty_ci", "fisher_alpha", '
-         '"berger_parker_d", "heip_e", "simpson_e", "dominance", '
-         '"mcintosh_e", "lladser_pe", "gini_index", "faith_pd"]'),
-        'observed_otus'],
-    'i-tree': ['choice:["default", "None"]', 'None']}
+    'Diversity metric': [
+        'choice:%s' % dumps(list(ALPHA_DIVERSITY_METRICS)),
+        'Number of distinct features'],
+    'Phylogenetic tree': ['choice:["default", "None"]', 'None']}
 outputs = {'o-alpha-diversity': 'alpha_vector'}
 dflt_param_set = {
     'Defaults': {
-        'p-metric': 'observed_otus',
-        'i-tree': 'None'}
+        'Diversity metric': 'Number of distinct features',
+        'Phylogenetic tree': 'None'}
 }
 qiime_cmd = QiitaCommand(
-    "alpha_diversity", "Alpha Diversity",
+    "Calculate alpha diversity", "Alpha Diversity",
     alpha_diversity, req_params, opt_params, outputs, dflt_param_set,
     analysis_only=True)
 plugin.register_command(qiime_cmd)
 
 # Define the alpha_correlation command
-req_params = {'i-alpha-diversity': ('artifact', ['alpha_vector'])}
-opt_params = {'p-method': ['choice:["spearman", "pearson"]', 'spearman']}
+req_params = {'Alpha vectors': ('artifact', ['alpha_vector'])}
+opt_params = {'Correlation method':
+              ['choice:%s' % dumps(list(ALPHA_CORRELATION_METHODS)),
+               'Spearman']}
 outputs = {'q2_visualization': 'q2_visualization'}
-dflt_param_set = {
-    'Defaults': {
-        'p-method': 'spearman'}
-}
+dflt_param_set = {'Defaults': {'Correlation method': 'Spearman'}}
 qiime_cmd = QiitaCommand(
-    "alpha_correlation", "Alpha Corrrelation",
+    "Calculate alpha correlation", "Alpha Corrrelation",
     alpha_correlation, req_params, opt_params, outputs, dflt_param_set,
     analysis_only=True)
 plugin.register_command(qiime_cmd)
 
 # Define the taxa barplot command
-req_params = {'i-table': ('artifact', ['BIOM'])}
+req_params = {'BIOM table': ('artifact', ['BIOM'])}
 opt_params = {}
 outputs = {'q2_visualization': 'q2_visualization'}
 dflt_param_set = {
     'Defaults': {}
 }
 qiime_cmd = QiitaCommand(
-    "taxa_barplot", "Taxa Barplot",
+    "Summarize taxa", "Taxa Barplot",
     taxa_barplot, req_params, opt_params, outputs, dflt_param_set,
     analysis_only=True)
 plugin.register_command(qiime_cmd)
 
 # Define the filtering samples from biom command
-req_params = {'i-table': ('artifact', ['BIOM'])}
+req_params = {'BIOM table': ('artifact', ['BIOM'])}
 opt_params = {
-    'p-min-frequency': ('integer', 0),
-    'p-max-frequency': ('integer', 9223372036854775807),
-    'p-min-features': ('integer', 0),
-    'p-max-features': ('integer', 9223372036854775807),
-    'p-where': ('string', '')}
+    'Minimum feature frequency across samples': ('integer', 1),
+    'Maximum feature frequency across samples':
+        ('integer', 9223372036854775807),
```

Comment:
Is there a more polite / less typo-able way to represent max 64 bit signed integer value?

Reply:
@antgonza you used this value. Is there any specific reason you used that value instead of using `sys.maxint`?

Reply:
Thanks for confirming - removed the hardcoded value